packages/sdk/src/syncedStreamsLoop.ts

```diff
@@ -548,6 +548,11 @@ export class SyncedStreamsLoop {
     }
 
     private async tick() {
+        const item = this.responsesQueue.shift()
+        if (item) {
+            await this.onUpdate(item)
+        }
```

Comment:
There is a throw in this function that can trigger now. Can you remove it? ![image](https://github.com/user-attachments/assets/3b6cecef-77f2-4ca6-9aea-86a281fcec5c)